web-access-app/src/main/resources/recordTransformations/modsToFullRecord.xsl

```diff
@@ -920,11 +949,19 @@
 								<xsl:with-param name="field" select="*[local-name() = 'note' and not(@type = 'admin')]"/>
 							</xsl:call-template>
 
-							<xsl:call-template name="modsGroupedFieldWithType">
-								<xsl:with-param name="defaultLabel">Identifier</xsl:with-param>
-								<xsl:with-param name="field" select="*[local-name() = 'identifier']"/>
-							</xsl:call-template>
+							<xsl:variable name="collectionIdentifier" select="*[local-name() = 'identifier' and @displayLabel='Collection Number']"/>
+							<xsl:variable name="identifier" select="*[local-name() = 'identifier' and not(@displayLabel='Collection Number')]"/>
+							<xsl:if test="boolean($collectionIdentifier) or boolean($identifier)">
```

Comment:
Is the conditional wrapping both of these template calls needed? It seems like typically the templates don't display anything if the the selector didn't return any elements. Did the display look weird without it?

Reply:
It seems to work okay if I remove the variables and the conditional...I think I was just trying to make it more similar to the code in the other section. I will remove and update